### PR TITLE
pt_PT layout support added

### PR DESCRIPTION
**Pull request by Samega7Cattac**
Note: To do the spaces between the F keys needed to add a key without text and disabled since the `null` key have a static width

### CODE REVIEW

**Comment by Samega7Cattac** [comment]
Tested on Razer Blackwindow chroma v2

File: data/matrix_layouts/razerdefault22.json [hunk 1]
```diff
@@ -539,5 +539,142 @@
...
+            {"label": "F6", "matrix": [0, 8]},
+            {"label": "F7", "matrix": [0, 9]},
+            {"label": "F8", "matrix": [0, 10]},
+            {"label": "", "disabled": true, "width": 27},
+            {"label": "F9", "matrix": [0, 11]},
+            {"label": "F10", "matrix": [0, 12]},
+            {"label": "F11", "matrix": [0, 13]},
+            {"label": "F12", "matrix": [0, 14]},
+            {"label": "prt sc\nsys rq", "matrix": [0, 15]},
+            {"label": "scr\nlk", "matrix": [0, 16]},
```
Comment (z3ntu) [inline data/matrix_layouts/razerdefault22.json]: this label should be in one line

**Comment by z3ntu** [comment]
Generally looks very good!

Can you please make the enter button in the third row enabled and the one in the fourth disabled? Also, you can remove the matrix attributes for disabled buttons as they are not used :)

**Comment by Samega7Cattac** [comment]
I tested with only third row enter and it didn't assign any color to it

**Comment by z3ntu** [comment]
If you put the matrix attribute from the fourth row to the button in the third row it should work

**Comment by Samega7Cattac** [comment]
u right, done

**Comment by Samega7Cattac** [comment]
is something missing to accept this PR?

**Comment by z3ntu** [comment]
Everything's fine here, I just kind of forgot. Thanks for the PR!

**Pull request merged**